main/guides/getting-started/assets/zoe-simp.mmd

```diff
@@ -9,6 +9,6 @@ end
     agent->>+zoe: offer(invitation, {give, want}, pmts)
     zoe-->>agent: userSeat
     zoe->>+contract: {give, want}
-    contract-->>-zoe: reallocate()
+    contract-->>-zoe: atomicRearrange()
```

Comment:
the corresponding .svg should be regenerated.

it shows up in _Specifying Offers_

https://10871-excisereallocate.documentation-7tp.pages.dev/guides/getting-started/contract-rpc.html#specifying-offers

https://10871-excisereallocate.documentation-7tp.pages.dev/assets/zoe-simp.1Ml6HzzR.svg

Crud: CONTRIBUTING.md doesn't say how. IOU, I suppose.
In brief: I use [Mermaid Editor \- Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=tomoyukim.vscode-mermaid-editor).

Reply:
That page also has a link to a web editor which works fine for small diagrams like this, and allowed me to save as svg.